src/pages/MembersPage/LeaveCircleModal.tsx

```diff
@@ -110,7 +110,7 @@ export const LeaveCircleModal = ({
               color="destructive"
               size="large"
               css={{ width: '50%' }}
-              disabled={!isEmpty(errors) || !isDirty}
+              disabled={!isValid}
```

Comment:
@sshmm FYI look at the changes/improvements @wingfeatherwave and I made here. For example, !isValid is a simpler way of doing what you did before.

Reply:
Got it thanks